styles/styles.css

```diff
@@ -477,5 +495,114 @@ main .section.our-history .year.highlight .icon-calendar {
   main .section.our-history .year:last-of-type::after {
     height: 85%;
   }
+}
+
+/* STYLED SECTIONS */
+
+/* stylelint-disable-next-line no-descending-specificity */
+.section.inverted-text,
+.section.inverted-text h2,
+.section.inverted-text h3 {
+  color: var(--mt-global-color-white);
```

Comment:
please don't use base color vars directly, but instead `--mt-background-color-primary` for example.

Reply:
DONE